src/GTP.cpp

```diff
@@ -243,6 +243,146 @@ std::string GTP::get_life_list(const GameState & game, bool live) {
     return result;
 }
 
+AnalyzeTags GTP::parse_analyze_tags(std::istringstream & cmdstream, const GameState & game) {
+    std::string tag;
```

Comment:
Use auto for consistency with the rest of the code.

Reply:
Just asking to make sure I understand you correctly as I am a bit puzzled here: You are referring to the variables in the function body below this line, and not to the `tag` variable or the return value of the function, right? The rest of the codebase does not seem to use `auto` for return types.

Anyway, I think I'll turn the function into a constructor or a method of `AnalyzeTags` when I make the members of the class private as you requested.